packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -439,62 +454,62 @@ class FlexLayoutProperties extends LayoutProperties {
   }
 
   Axis get crossAxisDirection {
-    return direction == Axis.horizontal ? Axis.vertical : Axis.horizontal;
+    return direction;
```

Comment:
revert this change. We need to return the opposite direction for crossAxisDirection

Reply:
done